language/fixtures/args_forwarding.rb

```diff
@@ -0,0 +1,9 @@
+class F
```

Comment:
Could you nest this under `DelegationSpecs` and name it more explicitly like `Target`?
That will also fix the CI.